llvm/test/tools/gold/X86/unified-lto.ll

```diff
@@ -0,0 +1,42 @@
+; Check that we can use full LTO with gold plugin when inputs
```

Comment:
Can you add some kind of check that unified LTO inputs works with ThinLTO as well?